t/run/todo.t

```diff
@@ -33,7 +31,10 @@ TODO: {
         "abcde5678" =~ / b .* (*plb:(*plb:(.{4}))? (.{5}) ) .$ /x;
         print $1 // "undef", ":", $2 // "undef", "\n";
         EOF
-    "undef:de567\nundef:de567", { $switches }, "");
+        "undef:de567\nundef:de567",
+        {}, # Anonymous hash must have even number of elements
```

Comment:
This comment looks weird to me in isolation. It feels more like a commit message (for this particular change) than a perpetual annotation.

But apart from that, looks good to me.

Reply:
I can delete that comment -- but let's hear first from @khwilliamson and others.

Reply:
I don't know how this slipped in there.  The code in question was meant to be `{ eval $switches }`, which is in use in other test files, such as `t/run/locale.t`